plugin/coala.vim

```diff
@@ -63,5 +63,3 @@ function! coala#run()
 endfunction
 
 au BufWritePost * call s:WritePostHook()
```

Comment:
don't we leave a newline at the end of the files? i thought thats how pep8 works.